src/main/java/com/sourcegraph/javagraph/ClassPathUtil.java

```diff
@@ -0,0 +1,63 @@
+package com.sourcegraph.javagraph;
+
+import com.google.common.io.Files;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+public class ClassPathUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClassPathUtil.class);
+
+    public static Collection<String> transformClassPath(Collection<String> classPath) {
+        Collection<String> ret = new ArrayList<>();
+        for (String classPathElement : classPath) {
+            ret.addAll(processClassPathElement(classPathElement));
+        }
+        return ret;
+    }
+
+    private static Collection<String> processClassPathElement(String classPathElement) {
+        File file = new File(classPathElement);
+        if (!file.isFile() || !file.getName().endsWith(".aar")) {
+            return Collections.singletonList(classPathElement);
+        } else {
+            return unpackAar(file);
+        }
+    }
+
+    private static Collection<String> unpackAar(File aarFile) {
+        // TODO: use environment variable that points to SRCLIB temp directory to keep files there
```

Comment:
removed this file in alexsaveliev/srclib-java@0b25677b36904e61ee0dd0eb1a4796d45f83613a